pkg/apis/sources/v1alpha1/githubsource_types.go

```diff
@@ -31,6 +31,7 @@ var _ runtime.Object = (*GitHubSource)(nil)
 var _ = duck.VerifyType(&GitHubSource{}, &duckv1alpha1.Conditions{})
 
 // GitHubSourceSpec defines the desired state of GitHubSource
+// +kubebuilder:categories=all,knative,eventing,sources
```

Comment:
I didn't know about this annotation; thanks @matzew for discovering it!

Reply:
@grantr I saw in other code ;-)